e2e-tests/functions

```diff
@@ -744,6 +744,10 @@ spinup_pxc() {
 		apply_config "$config"
 	fi
 
+	# todo(remove after 1.10 release)
+	sleep 10
+	kpatch_set_field svc "${cluster}-pxc-unready" "/spec/publishNotReadyAddresses" true
+
```

Comment:
can we add the change only `if k8s version >= 1.21 and platform is EKS` condition?